app/assets/javascripts/ui_components/active_buttons.coffee

```diff
@@ -0,0 +1,5 @@
+$ ->
+  $('.btn-group .btn').click ->
+    if $(this).siblings().hasClass 'active'
```

Comment:
@RKushnir it's true - i guess i was thinking about performance, having it fail earlier and listen to fewer buttons. but it probably doesn't make much of a difference. i could remove the condition?